ui-tests/src/test/java/org/openmrs/reference/page/MarkPatientDeceasedPage.java

```diff
@@ -0,0 +1,118 @@
+    private static final By CONCEPT_NAME = By.id("namesByLocale[en].name");
+    private static final By ADD_ANSWER = By
+            .cssSelector("#codedDatatypeRow > td > table > tbody > tr > td.buttons > input:nth-child(1)");
+    private static final By CAUSEBOX = By.id("newAnswerConcept");
+    private static final By FEVER = By.xpath("/html/body/ul[1]/li[2]/a");
+    private static final By ADD_BUTTON = By.xpath("/html/body/div[5]/div[11]/button[1]");
+    private static final By SAVE = By.xpath("/html/body/div[1]/div[3]/div[2]/form/div/input[1]");
+    private static final By SEARCH = By.id("inputNode");
+    private static final By COD = By.xpath("//*[@id=\"openmrsSearchTable\"]/tbody/tr[1]/td/span");
+    private static final By NEW_COD_ID = By.xpath("//*[@id=\"conceptTable\"]/tbody/tr[1]/td");
+    private static final By ADMINISTRATION = By.id("administrationNavLink");
+    private static final By ADVANCED_SETTINGS = By.linkText("Advanced Settings");
+    private static final By CONCEPT_VALUE = By.xpath("//*[@id=\"globalPropsList\"]/tr[133]/td[2]/input");
+    private static final By SAVE_CONCEPT_VALUE = By.xpath("//*[@id=\"buttonsAtBottom\"]/input[1]");
+    private static final By HOME = By.id("homeNavLink");
+    private static final By MARK_PATIENT_DECEASED = By.id("org.openmrs.module.coreapps.markPatientDead");
+    private static final By CHECK_BOX = By.id("deceased");
+    private static final By DATE_PICKER = By.id("death-date-display");
+    private static final By SAVE_DECEASED = By
+            .cssSelector("#mark-patient-dead > fieldset > p:nth-child(4) > span:nth-child(2) > input");
+    private static final By ACTIVE_DAY = By.cssSelector("td.day.active");
+    private static final By DEATH_MESSAGE = By.className("death-message");
+    private static final By EDIT = By.id("editConcept");
+    private static final By DELETE_BUTTON = By.cssSelector("#saveDeleteButtons > input[type=submit]:nth-child(4)");
+
+    public MarkPatientDeceasedPage(Page page) {
+        super(page);
+    }
+
+    public void getConcept() {
+        findElement(DICTIONARY).click();
+        Concept();
+        findElement(HOME).click();
+    }
+
+    /*
+     * If there is a concept called Cause Of Death, The method gets its concept code
+     * if there is no concept at all or no concept called Cause of death, the method
+     * creates one and copies its concept code
+     */
+    public void Concept() {
+        newConcept();
+        getConceptid(newId());
+    }
+
+    public void clickOnMarkPatientDead() {
+        findElement(MARK_PATIENT_DECEASED).click();
+        findElement(CHECK_BOX).click();
+        findElement(DATE_PICKER).click();
+        pickDate();
+        Select dropdown = new Select(findElement(By.id("cause-of-death")));
+        dropdown.selectByValue("140238AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA");
+        findElement(SAVE_DECEASED).click();
+    }
+
+    public void newConcept() {
```

Comment:
Rename the method to a  more readable name!

Reply:
@insookwa did you forget to catch the previous [comment ](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/529/files#r701124946) !